spec/provider/code_deploy_spec.rb

```diff
@@ -89,7 +89,9 @@
       s3_location: {
         bucket:      'bucket',
         bundle_type: 'tar',
-        key:         'key'
+        key:         'key',
+        version:     'ObjectVersionId',
```

Comment:
Use `object_version_id`, and `etag`. The names are inconsequential here, so lower the cost of skimming the code by mimicking other constants in this bit.